src/yang/example-compa-diffserv.yang

```diff
@@ -32,60 +32,73 @@ module example-compa-diffserv {
      companyA diffserv specification extension.";
   revision YYYY-MM-DD {
     description
-      "Initial revision for diffserv actions on network packets";
+      "Initial revision for diffserv actions on network packets.";
     reference
       "RFC XXXX: A YANG Data Model for Quality of Service (QoS).";
   }
 
   identity default-policy-type {
     base qos-types:policy-type;
     description
-      "This defines default policy-type";
+      "This defines default policy-type.";
   }
 
   identity qos-group {
     base qos-types:filter-type;
     description
-      "qos-group filter-type";
-  }
-
-  identity drop {
-    base qos-types:action-type;
-    description
-      "drop action-type";
+      "Filter-type to match on  packet metadata.";
   }
 
   grouping qos-group {
     list qos-group {
-      key "qos-group-min qos-group-max";
+      key "min max";
       description
-        "list of dscp ranges";
-      leaf qos-group-min {
+        "List of qos-group ranges.";
+      leaf min {
         type uint8;
         description
-          "Minimum value of qos-group range";
+          "Minimum value of qos-group range.";
       }
-      leaf qos-group-max {
+      leaf max {
         type uint8;
+        must ". >= ../min" {
+          error-message
+          "The qos-group max must be 
+           greater than or equal to qos-group min.";
+        }
         description
-          "Maximum value of qos-group range";
+          "Maximum value of qos-group range.";
       }
     }
     description
-      "Filter containing list of qos-group ranges";
+      "Filter containing list of qos-group ranges.";
   }
-  grouping drop {
-    container drop {
-      leaf drop {
-        type empty;
+  grouping bw-remaining-percent {
+    container bw-remaining-percent {
+      leaf value {
+        type uint8;
         description
-          "drop action";
+          "Value for the rate that is configured.";
```

Comment:
What value is this? Reserved and priority bandwidth? Or what is left over? What are its units?

Reply:
what is left over. Unit is percentage

Reply:
Then it is not a value. It is a percentage of the remaining bandwidth.